app/static/js/custom-dropdown.js

```diff
@@ -0,0 +1,37 @@
+document.addEventListener("DOMContentLoaded", () => {
+  const allSections = document.querySelectorAll("section")
+  const sectionIds = Array.apply(null, allSections).map((section) =>
+    section.getAttribute("id"),
+  )
```

Comment:
Tiny thing: I would prefer `Array.from()` here, which does not use mystery parameters like `null` here. :D 

```suggestion
  const sectionIds = Array.from(allSections).map((section) =>
    section.getAttribute("id"),
  )
```